docs/manufacturer/manufacturer-design-guidelines.mdx

```diff
@@ -265,6 +265,32 @@ Practically, this means that all pins should be on the same port, or at most two
 
   As an additional reference design, see the Fenix F405: https://oshwlab.com/jyesmith/fenix-f405
 
+### 3.2.2 Select appropriate default UARTs to avoid hijacking USB DFU 
+
+When selecting UARTs, for the default RX connection or to route to JST sockets, care should be taken to avoid those MCU pins which may interfere with USB DFU.
+
+The DFU protocol will attach to any MCU UART which is sending traffic, not only the USB VCP connection. This is a design feature of STM32 F4 MCUs and cannot be disabled.
```

Comment:
Pedantic point: the USB isn't "VCP", it's "DFU", which is a separate protocol and doesn't simulate a serial port.

Reply:
> Pedantic point: the USB isn't "VCP". DFU is a separate protocol and doesn't simulate a serial port.

What is the correct term? I saw mention of OTGFS or similar with ArteryTek

Reply:
The correct term is just USB DFU.
DFU (device firmware upgrade) it s USB protocol, in the same way that VCP (virtual com port) is.